monitor/rpcmonitor/rpc.go

```diff
@@ -218,7 +222,12 @@ func DefaultRPCMetadataExtractor(event *EventInfo) (*policy.PURuntime, error) {
 }
 
 func validateEvent(event *EventInfo) error {
-
+	zap.L().Error("EventInfo", zap.String("EventInfo.PUID", event.PUID),
+		zap.String("EventInfo.PID", event.PID),
+		zap.String("EventType", string(event.EventType)),
+		zap.Bool("HostService", event.HostService),
+		zap.String("CGROUP", event.Cgroup),
+	)
 	if event.EventType == monitor.EventCreate || event.EventType == monitor.EventStart {
 		if len(event.Name) > 64 {
 			return fmt.Errorf("Invalid Event Name - Must not be nil or greater than 32 characters")
```

Comment:
the check and the error is not coherent. you check for >64 and you say it must be <=32